etna/transforms/decomposition/detrend.py

```diff
@@ -164,11 +166,16 @@ def __init__(self, in_column: str, poly_degree: int = 1, **regression_params):
                 in_column=self.in_column,
                 regressor=LinearRegression(**self.regression_params),
                 poly_degree=self.poly_degree,
-            )
+            ),
+            required_features=[self.in_column],
         )
 
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform."""
+        return []
```

Comment:
Why isn't it regressor? As I understand it can predict values in the future.

Reply:
This method return the regressors CREATED by this transform, but in_column was already regressor before this transform, so it does not create the new regressor.